testsuite/features/build_validation/retail/proxy_container_branch_network.feature

```diff
@@ -0,0 +1,86 @@
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Activate the branch network on the proxy
+    When I activate the private network on the proxy
+
+  Scenario: Check the DHCP server works as expected
+    Then the "dhcp_dns" host should be present on private network
+    And name resolution should work on private network
+
+  Scenario: The DHCP server should not reach the server
+    Then the "dhcp_dns" host should not communicate with the server using private interface
+
+  Scenario: Check that the proxy knows about the private network
+    When I follow "Details" in the content area
+    And I follow "Hardware" in the content area
+    Then I should see a "proxy.example.org" text
+
+  Scenario: Create branch terminals group
+    When I follow the left menu "Systems > System Groups"
+    And I follow "Create Group"
+    And I enter "example" as "name"
+    And I enter "Terminal branch: example.org" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group example created." text
+    When I follow "Target Systems"
+    And I check the "proxy" client
+    And I click on "Add Systems"
+    Then I should see a "1 systems were added to example server group." text
+
+  Scenario: Create all terminals group
+    When I follow the left menu "Systems > System Groups"
+    When I follow "Create Group"
+    And I enter "TERMINALS" as "name"
+    And I enter "All terminals" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group TERMINALS created." text
+
+  Scenario: Create all branch servers group
+    When I follow the left menu "Systems > System Groups"
+    When I follow "Create Group"
+    And I enter "SERVERS" as "name"
+    And I enter "All branch servers" as "description"
+    And I click on "Create Group"
+    Then I should see a "System group SERVERS created." text
+    When I follow "Target Systems"
+    And I check the "proxy" client
+    And I click on "Add Systems"
+    Then I should see a "1 systems were added to SERVERS server group." text
+
+  Scenario: Parametrize TFTP on the branch server
+    When I follow "Formulas" in the content area
+    And I follow first "Tftpd" in the content area
+    And I click on "Expand All Sections"
+    And I enter the local IP address of "proxy" in internal network address field
+    And I enter "/srv/saltboot" in TFTP base directory field
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
+
+  Scenario: Parametrize vsFTPd on the branch server
+    When I follow "Formulas" in the content area
+    And I follow first "Vsftpd" in the content area
+    And I click on "Expand All Sections"
+    And I enter the local IP address of "proxy" in vsftpd internal network address field
+    And I enter "/srv/saltboot" in FTP server directory field
+    And I click on "Save Formula"
+    Then I should see a "Formula saved" text
```

Comment:
We do not use FTP anymore, this is not needed even on normal branch deployment.

Reply:
I'm not here yet. It is a copy and paste for now.